Library/Adjustment.cs

```diff
@@ -299,14 +299,15 @@ internal void WriteXml(XmlTextWriter xmlWriter, bool embedded = false)
             xmlWriter.WriteStartElement("adjustment"); // Start: adjustment
             xmlWriter.WriteElementString("unit_amount_in_cents", UnitAmountInCents.AsString());
             xmlWriter.WriteElementString("quantity", Quantity.AsString());
-            xmlWriter.WriteElementString("item_code", ItemCode);
```

Comment:
This is from https://github.com/recurly/recurly-client-net/pull/477 and somehow ended up here bc I didn't git right.